app/controllers/editions_controller.rb

```diff
@@ -24,6 +24,19 @@ def new
     @work = Work.new
   end
 
+  def import
+    steam_url = params.permit(:steam_url)[:steam_url]
+    if steam_url
+      @edition = SteamImporterService.import_edition(url: steam_url)
+      work = Work.new(
+        original_title: @edition.title,
+        original_release_date: @edition.release_date
+      )
+
+      create_with_new_work(work: work)
```

Comment:
instead of passing the whole work to this function, I think it would make more sense to pass the params instead.

Reply:
The problem is that I don't have the `params` in the `import` method. I can create a fake object for that.. do you think it's better?

Reply:
Can't you pass the two needed params instead?

Reply:
Hm.. yes

Reply:
Done